### DISCUSSION TITLE
will it run on these specs?

### DISCUSSION THREAD

**Discussion by kpoxo6op** [onedr0p/cluster-template#1576] [General]
hello,

will I be able to create a working cluster on these 3 miniPCs:
```
CPU: Intel Celeron N3350 (Dual-Core, 1.10GHz base frequency, up to 2.4GHz)
RAM: 6GB
Storage: 64GB ROM (expandable via M.2 SATA 2280 SSD)
```

I am just getting started, all I need is to create the cluster, I won't run any workloads at this stage. Thanks!

**Comment by onedr0p** [comment]
I don't see why not, but you'll be limited to how many workloads can run on it due to the limited resources.

**Reply by kpoxo6op** [reply]
Thanks for confirming!

I was not sure because My miniPCs specs are way below Talos [requirements](https://github.com/onedr0p/cluster-template?tab=readme-ov-file#system-requirements). They match  [kubespray](https://github.com/kubernetes-sigs/kubespray?tab=readme-ov-file#requirements) requirements though.

I am more keen start with Talos because it looks more Infra-as-code to me.

**Reply by onedr0p** [reply]
Yeah they are really low spec but you won't really know until you try.
